### DISCUSSION TITLE
Original LED charging behavior.

### DISCUSSION THREAD

**Discussion by Crispin777** [nefarius/DsHidMini#291] [Ideas]
It would be nice if there was an option to have original DualShock 3 LED indicator behavior (i.e. blinking player 1 indicator or whichever player it is switched to).

**Comment by nefarius** [comment]
This is not possible, Windows has not concept of a player number, it all depends on the games and which input APIs they use.

**Reply by Crispin777** [reply]
Uh. i meant to make it so that only one indicator blinks whilst charging, the way it was originally.

**Reply by nefarius** [reply]
In v3 - if I ever find the time again to work on that - it is possible to customize the charging animation. For now, it is what it is.
